src/sentry/static/sentry/app/views/organizationIntegrations/integrationItem.jsx

```diff
@@ -41,9 +34,45 @@ export default class IntegrationItem extends React.Component {
               </Tooltip>
             )}
           </IntegrationName>
-          <DomainName style={style}>{integration.domainName}</DomainName>
-        </Box>
+          <DomainName>{integration.domainName}</DomainName>
+        </Labels>
       </Flex>
     );
   }
 }
+
+const IntegrationName = styled.div`
+  font-size: 1.6rem;
+`;
+
+const DomainName = styled.div`
+  color: ${p => p.theme.gray3};
+  font-size: 1.4rem;
+`;
+
+const ExpandedLabels = styled(p => <Box pl={1} {...p} />)`
+  display: flex;
+  flex-direction: column;
+
+  /* stylelint-disable-next-line no-duplicate-selectors */
+  ${DomainName} {
```

Comment:
Can we move this into `DomainName`?

e.g. `${ExpandedLabels} & {...}`

Reply:
5d0b991